pkgs/development/tools/build-managers/bazel/bazel_3/default.nix

```diff
@@ -550,7 +550,7 @@ stdenv.mkDerivation rec {
     mkdir $out/share
     cp ./bazel_src/bazel-bin/src/tools/execlog/parser_deploy.jar $out/share/parser_deploy.jar
     mv ./bazel_src/output/bazel $out/bin/bazel-${version}-${system}-${arch}
-    cat <<EOF > $out/bin/execlog
+    cat <<EOF > $out/bin/bazel-execlog
     #!${runtimeShell} -e
     ${runJdk}/bin/java -jar $out/share/parser_deploy.jar \$@
     EOF
```

Comment:
you also have to change the name in the chmod call

Reply:
I am ashamed ! sorry :'( fixed !